physionet-django/events/views.py

```diff
@@ -310,3 +314,46 @@ def event_detail(request, event_slug):
          'is_waitlisted': is_waitlisted,
          'event_datasets': event_datasets,
          })
+
+
+@require_http_methods(["PUT", "DELETE", "GET", "POST"])
+def manage_co_hosts(request, event_slug):
+    """
+    Manage co-hosts of an event
+    """
+    user = request.user
+    error_message = None
+
+    body = json.loads(request.body)
```

Comment:
@Rutvikrj26 here's the line that parses the body. It parses it regardless of the HTTP method, even though the views specified the following: `["PUT", "DELETE", "GET", "POST"]`.